sormas-api/src/main/resources/strings.properties

```diff
@@ -986,6 +986,7 @@ messageImportPartiallySuccessful = <b>Import partially successful!</b><br/>The i
 messageImportSuccessful = <b>Import successful!</b><br/>All rows have been imported. You can now close this window.
 messageUploadSuccessful = Upload successful! You can now close this window.
 messageIncompleteGpsCoordinates = GPS coordinates are incomplete
+messageLabMessagesAssigned = All selected lab messages have been (un)assigned
```

Comment:
`(un)assigned` could be misleading, I think it would be better something like `The assignee has been changed for all selected lab messages`